app/scripts/controllers/progressBar.js

```diff
@@ -0,0 +1,36 @@
+angular.module('admissionSystemApp')
+  .controller('ProgressBarCtrl', ['$scope', 'progressBarService', function ($scope, progressBarService) {
+
+
+    $scope.status = "Заповнено на: ";
+
+    $scope.value = progressBarService;
+
+    $scope.$watch('value.value', function (value) {
```

Comment:
Function can be the first parameter of $watch expression, so, I think, there is no need to save progressBarService into $scope.value if we don't use it anywhere